specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -0,0 +1,247 @@
+|     static   event EventHandler<object> EnergySaverStatusChanged         |     Event   raised when the EnergySaverStatus changes.                                                                  |
+|     static   event EventHandler<object> PowerSupplyStatusChanged         |     Event   raised when the PowerSupplyStatus changes.                                                                  |
+|     static   event EventHandler<object> RemainingChargePercentChanged    |     Event   raised when the RemainingChargePercent changes.                                                             |
+|     static   event EventHandler<object> RemainingDischargeTimeChanged    |     Event   raised when the RemainingDischargeTime changes.                                                             |
+|     static   PowerSourceStatus PowerSourceStatus                         |     Readonly   property, gets the device's current power source.                                                        |
+|     static   event EventHandler<object> PowerSourceStatusChanged         |     Event   raised when the system power source has changed.                                                            |
+|     static   DisplayStatus DisplayStatus                                 |     Readonly   property, gets the current display state.                                                                |
+|     static   event EventHandler<object>  DisplayStatusChanged            |     Event   raised when the current display state has changed.                                                          |
+|     static   SystemIdleStatus SystemIdleStatus                           |     Readonly   property, gets the current system idle status.                         |
+|     static   event EventHandler<object> SystemIdleStatusChanged          |     Event   raised when the system's idle state has changed.                          |
+|     static   PowerSchemePersonality PowerSchemePersonality               |     Readonly   property, gets the active power scheme personality.                    |
+|     static   event EventHandler<object> PowerSchemePersonalityChanged    |     Event   raised when the active power scheme personality has changed.              |
+|     static   UserPresenceStatus UserPresenceStatus                       |     Readonly   property, gets the user's present/absent status.                       |
+|     static   event EventHandler<object> UserPresenceStatusChanged        |     Event   raised when the user's presence status has changed.                       |
+|     static   SystemAwayModeStatus SystemAwayModeStatus                   |     Readonly   property, gets the entering/exiting status of the system away mode.    |
+|     static   event EventHandler<object> SystemAwayModeStatusChanged      |     Event   raised when the system is entering or exiting away mode.                  |
+
+<br>
+<br>
+
```

Comment:
Do put a dump of the IDL of the new type, including its full namespace name, here. Just like we would have done in a product API spec.